MapboxCoreNavigation/RouteController.swift

```diff
@@ -675,6 +670,27 @@ extension RouteController: CLLocationManagerDelegate {
         }
     }
     
+    /**
+     Monitors the user's course to see if it is consistantly moving away from what we expect the course to be at a given point.
+     */
+    func userCourseIsOnRoute(_ location: CLLocation) -> Bool {
+        let nearByCoordinates = routeProgress.currentLegProgress.nearbyCoordinates
+        
+        guard let calculatedCourseForLocationOnStep = interpolatedCourse(from: location, along: nearByCoordinates) else { return true }
+        
+        let maxUpdatesAwayFromRouteGivenAccuracy = Int(location.horizontalAccuracy / Double(RouteControllerIncorrectCourseMultiplier))
```

Comment:
The horizontal accuracy pertains to the coordinates; it has nothing to do with the course. It makes little sense to divide a measurement in meters by a measurement in degrees.